mmv1/products/storage/terraform.yaml

```diff
@@ -72,7 +72,7 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         bucket_name: "static-content-bucket"
         object_name: "public-object"
     id_format: "{{bucket}}/{{object}}/{{entity}}"
```

Comment:
thinking out loud: `id` is only read during import, and we actually read `import_format` there instead. We don't need to annotate this line b/c we never need to recreate the `id` based on the template.

Reply:
I was also concerned that changing this line might be backwards-incompatible? but I'm not sure if that's true.